### PR TITLE
feat: add runtime modification of log level

### PR DESCRIPTION
**Pull request by kade-ddnkv**

### CODE REVIEW

File: cmd/core/display_utils.go [hunk 1]
```diff
@@ -3,14 +3,32 @@ package core
...
 	"github.com/wmnsk/go-pfcp/ie"
 	"github.com/wmnsk/go-pfcp/message"
 )
 
+func InitLogger() {
+	output := zerolog.ConsoleWriter{Out: os.Stdout, TimeFormat: "2006/01/02 15:04:05"}
+	log.Logger = zerolog.New(output).With().Timestamp().Logger()
+}
+
+func ConfigureLoggerLevel(loggingLevel string) error {
```
Comment (pirog-spb) [inline cmd/core/display_utils.go:22]: `ConfigureLoggerLevel` -> `SetLoggerLevel`

File: cmd/core/api.go [hunk 2]
```diff
@@ -36,6 +38,7 @@ func CreateApiServer(bpfObjects *ebpf.BpfObjects, pfcpSrv *PfcpConnection, forwa
 		v1.GET("config", DisplayConfig())
 		v1.GET("xdp_stats", DisplayXdpStatistics(forwardPlaneStats))
 		v1.GET("packet_stats", DisplayPacketStats(forwardPlaneStats))
+		v1.POST("edit_config", EditConfig)
```
Comment (pirog-spb) [inline cmd/core/api.go:41]: If possible use `config` name for this request (but POST method). So we have `config` API for get config values. And the same method name for set config values.

File: cmd/core/display_utils.go (hunk 1 above)
Comment (kade-ddnkv) [reply cmd/core/display_utils.go]: Done

File: cmd/core/api.go (hunk 2 above)
Comment (kade-ddnkv) [reply cmd/core/api.go]: Done

**Comment by kade-ddnkv** [comment]
There is a small issue: when config.yml is edited, function OnConfigChange in main.go is called two times.

Seems that this problem is related to OS, which is firing an event multiple times. For instance, using ```touch -m config.yml``` from console fires only one event.

File: cmd/core/api.go [hunk 3]
```diff
@@ -60,6 +63,35 @@ func CreateApiServer(bpfObjects *ebpf.BpfObjects, pfcpSrv *PfcpConnection, forwa
 	return &ApiServer{router: router}
 }
 
+type EditConfigRequest struct {
```
[the remark this answers is not in the document]
Comment (kade-ddnkv) [reply cmd/core/api.go:66]: +?

**Review by pirog-spb — APPROVED** [review]

**Pull request merged**